packages/components/src/vue/components/feature-settings.vue

```diff
@@ -97,40 +97,41 @@
   </div>
 </template>
 <script>
-  import Setting from './setting.vue'
-  import Options from '@/lib/options/options.js'
-  import DependencyCheck from '@/vue/vuex-modules/support/dependency-check.js'
-  export default {
-    name: 'FeatureSettings',
-    // API modules that are required for this component
-    inject: {
-      app: 'app',
-      l10n: 'l10n',
-      settings: 'settings'
-    },
-    storeModules: ['app'], // Store modules that are required by this component
-    mixins: [DependencyCheck],
-    components: {
-      setting: Setting,
+import Setting from './setting.vue'
+import Options from '@/lib/options/options.js'
+import DependencyCheck from '@/vue/vuex-modules/support/dependency-check.js'
+export default {
+  name: 'FeatureSettings',
+  // API modules that are required for this component
+  inject: {
+    app: 'app',
+    l10n: 'l10n',
+    settings: 'settings'
+  },
+  storeModules: ['app'], // Store modules that are required by this component
+  mixins: [DependencyCheck],
+  components: {
+    setting: Setting
+  },
+  computed: {
+    mouseMoveChecked: function () {
+      return this.$store.state.app.mouseMoveOverrideUpdate && this.app.getMouseMoveOverride() ? 'true' : false
     },
-    computed: {
-      mouseMoveChecked: function() {
-        return this.$store.state.app.mouseMoveOverrideUpdate &&  this.app.getMouseMoveOverride() ? 'true' : false
-      },
-      featureOptions () {
-        return this.$store.state.settings.featureResetCounter + 1 ? this.settings.getFeatureOptions() : null
-      }
+    featureOptions () {
+      return this.$store.state.settings.featureResetCounter + 1 ? this.settings.getFeatureOptions() : null
+    }
+  },
+  methods: {
+    featureOptionChanged: function (name, value) {
+      const keyinfo = Options.parseKey(name)
+      this.settings.featureOptionChange(keyinfo.name, value)
+      this.app.applyFeatureOption(keyinfo.name)
```

Comment:
these always appear together. Should settings.featureOptionChange just call app.applyFeatureOption?

Reply:
`applyFeatureOption()` is used separately here: https://github.com/alpheios-project/alpheios-core/blob/dev-components-refactoring-7/packages/components/src/lib/controllers/app-controller.js#L1326. That's why it's a separate method. 
With having two different methods I wanted to separate responsibilities: `featureOptionChange()` is responsible for changing a value of the feature option within the settings controller while `applyFeatureOption()` - for applying an effect of that change to the UI from within the app controller. Does it make sense?

Reply:
ok, yes.